src/geovista/bridge.py

```diff
@@ -901,11 +925,8 @@ def from_unstructured(
         unstructured mesh. This is represented in terms of indices into the
         provided `xs` and `ys` mesh geometry.
 
-        Note that, the `connectivity` must define a mesh comprised of faces
-        based on the same primitive shape e.g., a mesh of triangles, or a mesh
-        of quads etc. Support is not yet provided for mixed face meshes. Also,
-        any optional mesh `data` provided must be in the same order as the mesh
-        face `connectivity`.
+        Note that any optional mesh `data` provided must be in the same order
+        as the mesh face `connectivity`.
```

Comment:
@trexfeathers ... or the order of the points, as data can be on points or cells 👍